Use replacement variables for dates limiting patients/visits.

njgraham opened the pull request:
- Removed unused i2b2loyalty_patients view - the SCILHS version remains commented out.

Commit:
Use replacement variables for dates limiting patients/visits.

 - Removed unused i2b2loyalty_patients view - the SCILHS version remains commented out.

dckc commented on Oracle/run-i2p-transform.sh:
```diff
@@ -16,6 +16,8 @@ set -e
 #export network_id=
 #export network_name=
 # export i2b2_etl_schema=
+# export min_pat_list_date_dd_mon_rrrr=
```
Maintaining this list is getting boring. But I don't suppose shell scripts are self-documenting w.r.t. this sort of interface stuff.

Actually, we could move this comment to code if we used a shell function. Worth the trouble?

Commit:
Use variable for the number of months in the past used to calculate enrollment.

 - The value is adjusted depending on whether or not we're running the transform against date-shifted data.
